spec/factories/drivers.rb

```diff
@@ -2,10 +2,11 @@
   factory :driver do
     name { FFaker::Name.name }
     license_number { FFaker::Identification.drivers_license }
-    mobile_number { FFaker::PhoneNumberSG.mobile_number }
+    mobile_number { '87654321' }
```

Comment:
oooo can we have a look into this, and see why ffaker is returning invalid mobile numbers occasionally?

Reply:
https://github.com/ffaker/ffaker/pull/46/files
not too sure why, does it include numbers that are out of service?

Reply:
hmm its very odd, lets merge this PR for now, since from what i can see the ffaker stuff should be generating a valid number, im not sure what the rules are inside the phone validator library though